backend/lib/calculate/adequacy.py

```diff
@@ -179,14 +178,18 @@ def calculate_adequacies(
             str(sum(len(locations) for locations in locations_to_check_by_point)))
     )
 
-    n_processors = config.get('number_of_adequacy_processors')
+    measurer = distance.get_measure(measure_name)
+    measurer_config = config.get('measurer_config')[measure_name]
+    executor_type = measurer_config['adequacy_executor_type']
+    n_processors = measurer_config['n_adequacy_processors']
 
-    with multiprocessing.Pool(processes=n_processors) as executor:
+    with executor_type(processes=n_processors) as executor:
```

Comment:
Why not simply `executor`? Or is it a protected name?

Reply:
I don't want to overload the name: 
```
with executor(8) as executor:
```